index/refresh.go

```diff
@@ -0,0 +1,28 @@
+package index
+
+import (
+	"github.com/kuzzleio/sdk-go/types"
+)
+
+func (i *Index) Refresh(index string) error {
+	if index == "" {
+		return types.NewError("Index.Refresh: index required", 400)
+	}
+
+	result := make(chan *types.KuzzleResponse)
+
+	query := &types.KuzzleRequest{
+		Controller: "index",
+		Action:     "refresh",
+		Index:      index,
+	}
+	go i.k.Query(query, nil, result)
+
+	res := <-result
+
+	if res.Error != nil {
```

Comment:
nitpicking: this test is unnecessary and res.Error can be returned directly, as this will end up as returning `nil` if there is no error

(this comment can be applied to any and all functions returning only an error object)